bootc/lamp/Containerfile

```diff
@@ -5,15 +5,11 @@ RUN dnf -y update; dnf install -y httpd mariadb mariadb-server php-fpm php-mysql
 ADD var var
 RUN systemctl enable httpd mariadb php-fpm
 
-# The following configures root user access.
-# Add ssh public keys to /user/etc-system/root.keys as is done below,
-# or, to give root user access via password "secure" (not recommended), uncomment the 2 lines below
-#RUN echo "root:secure" | chpasswd
-#
-#Substitute YOUR public key for the below-private key holder for the following public key will have root access
+# Substitute YOUR public key below-private key holder for the following public key will have root access
+# Alternatively, this section can be removed if configuring users with kickstart or bootc-image-builder
 RUN mkdir /usr/etc-system && \
     echo 'AuthorizedKeysFile /usr/etc-system/%u.keys' >> /etc/ssh/sshd_config.d/30-auth-system.conf && \
-    echo 'ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA2vfgIjZRMlDQ0ck5ub8S2sfgG1HTML3Y8lsA3yOz/UsVJY/vxnDVrLLZPNk3hLSAVP+W9+j+pF1XsnjQ6aVbrsSxuGkFQvJoZeK3EjZ0A55lqEvJPG9+IUi7rqHBh5yU/ZX9fQ6KkvmB2ECGR1CqOR1uednPJJ/7fHeTElycgOGlYT6hafuo5RV6O3GITA4VAKZdIE+0+N37p0Yej+vIVYG4iRZ7Jgk+ZnZlwyi8IG06CJfuGwpAw76c/rvI6GYYueFwfGjUxo474ReNn53GyBGnM9NtoBdWGFSiPeET7056kfV4TSNDMB24js8lrho3TqE3wl+szkBeKU3oAZ9okQ== dwalsh@localhost.localdomain' > /usr/etc-system/root.keys && chmod 0600 /usr/etc-system/root.keys
+    echo 'ssh-rsa AAAAB3.....' > /usr/etc-system/root.keys && chmod 0600 /usr/etc-system/root.keys
```

Comment:
This means the Demo will not work, we need to seed something to allow us to login to the System.  We should set this as a BUILD_ARG then we can pass it in at build time.  And just point at the public key in the users homedir that is running it.

Reply:
$ podman build --build-arg="SSHPUBKEY=$(cat $HOME/.ssh/id_rsa.pub)" /tmp/dan

ARG SSHPUBKEY
 echo $SSHPUBKEY > /usr/etc-system/root.keys && chmod 0600 /usr/etc-system/root.keys